src/tlo/core.py

```diff
@@ -94,16 +98,16 @@ def __init__(self, type_, description, categories=None):
             self.categories = categories
 
     @property
-    def python_type(self):
+    def python_type(self) -> Type[Any]:
```

Comment:
```suggestion
    def python_type(self) -> type:
```

[`Type[Any]` (and `type[Any]`) is equivalent to `type`](https://docs.python.org/3/library/typing.html#the-type-of-class-objects) and [`Type` is deprecated in from Python 3.9 onwards in favour of using `type` directly.](https://docs.python.org/3/library/typing.html#typing.Type)